leshan-core/src/test/java/org/eclipse/leshan/core/node/codec/LwM2mNodeEncoderTest.java

```diff
@@ -89,6 +89,15 @@ public void text_encode_multiple_instances() {
                 model);
     }
 
+    @Test
+    public void text_encode_opaque_as_base64_string() {
+        byte[] opaqueValue = new byte[] { 1, 2, 3, 4, 5 };
```

Comment:
Just a detail, as the spec talk about hex value maybe we should use hex notation?
```java
byte[] opaqueValue = new byte[] { 0x1, 0x2, 0x3, 0x4, 0x5 };
```